frontend/src/components/Navbar/Navbar.js

```diff
@@ -18,29 +19,52 @@ const MenuItems = [
 /**
  * Component produces a navigational bar that provides links to all the paths listed in MenuItems
  */
-class Navbar extends Component {
-  render() {
-    return(
-      <div>
-        <div className="logo-container">
-          <img src = "fsd_logo300.png" alt="feedLogo" id="logo"></img>
-        </div>
-        <nav className="NavbarItems">
-          <ul className='nav-menu'>
-            {MenuItems.map((item, index) => {
-              return (
-                <li key={index}> 
-                  <Link className={item.className} as={Link} to={item.url}>
-                    {item.title}
-                  </Link>
-                </li>
-              )
-            })}
-          </ul>
-        </nav>
-      </div>
-    );
+const Navbar = (props) => {
```

Comment:
Is there a reason why this was changed from a class component to a functional component?

Reply:
We changed it to a functional component because we wanted to use History from React Router Doms to allow redirection of pages when logout from the navbar is clicked.